### PR TITLE
Api calls

### PR DESCRIPTION
**Pull request by chrispalka**
@dominicsilvia or @adarchiev ready for your review

### CODE REVIEW

**Review by dominicsilvia — APPROVED** [review]
Code is approved as submitted

File: db-mongo/seed.js [hunk 1]
```diff
@@ -1,53 +1,54 @@
...
+  const url = `https://hipsum.co/api/?type=${type}&paras=${paras}`;
   fetch(url)
-  .then(response => response.json())
-  .then(data => {
-    data.forEach((x) => {
-        let productId = count < 10 ? `P00${count}` : count > 9 && count < 100 ? `P0${count}` : `P${count}`
+    .then((response) => response.json())
+    .then((data) => {
+      data.forEach((x) => {
+        const productId = count < 10 ? `P00${count}` : count > 9 && count < 100 ? `P0${count}` : `P${count}`;
```
Comment (nickfreemandesign) [inline db-mongo/seed.js:18]: @cpalka87 for double-ternaries i recommend breaking them into separate variables that explain what your logic is in words

File: package.json [hunk 2]
```diff
@@ -18,17 +20,22 @@
...
     "mongoose": "^4.11.6",
     "multer": "^1.4.2",
     "node-fetch": "^2.6.0",
+    "body-parser": "^1.19.0",
     "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
+    "webpack-merge": "^5.1.1"
```
Comment (nickfreemandesign) [inline package.json:38]: not sure this needs to be a dependency over dev-dependency. same for some other loaders and plugins

File: package.json [hunk 3]
```diff
@@ -39,8 +46,16 @@
...
+    "babel-preset-airbnb": "^5.0.0",
     "dotenv": "^8.2.0",
+    "eslint": "^7.6.0",
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-react": "^7.20.5",
+    "eslint-plugin-react-hooks": "^4.0.8",
     "style-loader": "^1.2.1",
     "underscore": "^1.10.2",
```
Comment (nickfreemandesign) [inline package.json:60]: if you use underscrore in your actual code, then likely it will break in production if not included in your dependencies

**Pull request merged**